app/models/concerns/interval.rb

```diff
@@ -20,6 +20,24 @@ module Interval
     scope :overlapping_with, ->(period) { where(*overlapping_with_range(period.started_on, period.finished_on)) }
     scope :containing_period, ->(period) { where(*containing_range(period.started_on, period.finished_on)) }
     scope :ongoing_on, ->(date) { where(*date_in_range(date)) }
+    scope :closest_to, ->(date) {
```

Comment:
We should add a comment explaining what this scope does for future devs reading this.